Source/WebKit/UIProcess/ios/WKContentViewInteraction.mm

```diff
@@ -11983,7 +12008,7 @@ - (void)setAsyncSystemInputDelegate:(id<UIAsyncTextInputDelegate>)delegate
 - (void)handleAsyncKeyEvent:(UIKeyEvent *)event withCompletionHandler:(void(^)(UIKeyEvent *, BOOL))completionHandler
 {
     auto webEvent = adoptNS([[::WebEvent alloc] initWithUIKeyEvent:event]);
-    [self handleKeyWebEvent:webEvent.get() withCompletionHandler:[originalEvent = retainPtr(event), completionHandler = makeBlockPtr(completionHandler)](::WebEvent *webEvent, BOOL handled) {
+    [self _internalHandleKeyWebEvent:webEvent.get() withCompletionHandler:[originalEvent = retainPtr(event), completionHandler = makeBlockPtr(completionHandler)](::WebEvent *webEvent, BOOL handled) {
```

Comment:
Is this a drive by fix?

Reply:
It's a necessary part of the patch; otherwise, any key event will instantly crash (since the non-legacy API currently calls into the legacy API).